webrender/src/renderer.rs

```diff
@@ -308,7 +308,7 @@ fn create_clear_shader(name: &'static str,
                        max_ubo_vectors: usize) -> ProgramId {
     let prefix = format!("#define WR_MAX_UBO_VECTORS {}", max_ubo_vectors);
 
-    let includes = &["shared_other"];
```

Comment:
That's what I did first, but then - `Device::create_program` requires a shared header, and so I decided to let it live for now :)